src/Sylius/Bundle/CoreBundle/Controller/ProductVariantController.php

```diff
@@ -54,4 +55,27 @@ public function updatePositionsAction(Request $request): Response
 
         return new JsonResponse();
     }
+
+    /**
+     * @return mixed
+     *
+     * @deprecated This function will be removed in Sylius 2.0, since Symfony 5.4, use explicit input sources instead
+     * based on Symfony\Component\HttpFoundation\Request::get
+     */
+    private function getParameterFromRequest(Request $request, string $key)
```

Comment:
I'm wondering if we shouldn't port it down to ResourceBundle

Reply:
it is temporary logic, I am not sure if we need to make bigger refactor of it.

Reply:
this "temporary" logic will be with us for a long time

Reply:
👍 for moving it to `ResourceBundle`. However, to not stop this PR for too long, I suggest we merge it and proceed with `ResourceBundle` changes parallelly 🖖